docs/nipoppy/overview.md

```diff
@@ -1,39 +1,90 @@
 ## What is Nipoppy (formerly mr_proc)? 
-*Process long and prosper*
 
----
+[*Process long and prosper*](https://en.wikipedia.org/wiki/Vulcan_salute)
 
-Nipoppy is a workflow manager for:
+[Nipoppy](https://github.com/neurodatascience/nipoppy) is a lightweight framework for analyzing (neuro)imaging and clinical data. It is designed to help users do the following:
 
-1. MRI and tabular data curation
-2. Standardized processing 
-3. Raw + processed data tracking
+1. Curate and organize data into a standard directory structure
+2. Run data processing pipelines in a semi-automated and reproducible way
+3. Track availability (including processing status, if applicable) of raw, tabular and derived data
+4. Extract imaging features from MRI derivatives data for downstream analysis
 
----
+### Example workflow
 
-## Modules
+Given a dataset to process, a typical workflow with Nipoppy can look like this:
 
-1. `Code`: Codebase [repo](code_org.md) for running and tracking workflows
-2. `Data`: [Dataset](data_org.md) organized in a specific directory structure
-3. `Containers`: Singularity containers encapsulating your processing pipelines
+1. Clone the [Nipoppy template code repository](https://github.com/neurodatascience/nipoppy)
```

Comment:
Do we want people to clone directly or first fork it for their dataset?

Reply:
Fork, updated